### PR TITLE
Finish TodoMVC example

### PR DESCRIPTION
**Pull request by krawaller**
Add all missing functionality in TodoMVC example, leverage the CSS.

Now it is feature complete!

image

Future work remaining:
* More comments throughout
* Tests

### CODE REVIEW

**Comment by krawaller** [comment]
Whoops, seems `localstorageAtom` uses the verifier even when there is no stored value. I wasn't expecting that! That's what caused the null state here, and detonated in poor Billy's face. 

I'll update #19 with a fix for that, and then bump `klyva` in this PR once that is merged and published.

File: examples/todo/components/TodoItem.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { Todo } from '../types'
+import { useSelector, atom } from 'klyva'
+import { RemovableAtom } from 'klyva/dist/types'
+import { TodoItemView } from './TodoItemView'
+import { TodoItemForm } from './TodoItemForm'
+
+type TodoItemProps = {
+  todoAtom: RemovableAtom<Todo, any>
```
Comment (merisbahti) [inline examples/todo/components/TodoItem.tsx]: You can use `RemovablePrimitiveAtom` here so, or in place of `any` put `SetState<Todo> `

File: examples/todo/components/TodoItemForm.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import { Todo } from '../types'
+import { focusAtom, useSelector } from 'klyva'
+import { RemovableAtom, PrimitiveAtom } from 'klyva/dist/types'
+import { SmartTextInput } from './SmartTextInput'
+
+type TodoItemFormProps = {
+  todoAtom: RemovableAtom<Todo, any>
```
Comment (merisbahti) [inline examples/todo/components/TodoItemForm.tsx]: Same as above comment:

> You can use `RemovablePrimitiveAtom` here so, or in place of `any` put `SetState<Todo> `

File: examples/todo/components/TodoItemView.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import { AtomCheckBox } from './AtomCheckBox'
+import { Todo } from '../types'
+import { focusAtom, useSelector } from 'klyva'
+import { RemovableAtom, PrimitiveAtom } from 'klyva/dist/types'
+
+type TodoItemViewProps = {
+  todoAtom: RemovableAtom<Todo, any>
```
Comment (merisbahti) [inline examples/todo/components/TodoItemView.tsx]: Aaaand same as above comment: 
> You can use `RemovablePrimitiveAtom` here so, or in place of `any` put `SetState<Todo> `

File: examples/todo/components/TodoItemForm.tsx (hunk 2 above)
Comment (krawaller) [reply examples/todo/components/TodoItemForm.tsx]: Ah! I meant to ask you about that but forgot. Thank you, will fix!

**Comment by merisbahti** [comment]
yeehaa 🤠 use that raw klyva 0.7 power!

**Comment by merisbahti** [comment]
ah, the perfect chrismas gift!! 🎁

**Comment by krawaller** [comment]
👌

**Pull request merged**